src/main/java/atas/storage/AtasStorage.java

```diff
@@ -5,42 +5,42 @@
 import java.util.Optional;
 
 import atas.commons.exceptions.DataConversionException;
-import atas.model.AddressBook;
-import atas.model.ReadOnlyAddressBook;
+import atas.model.ReadOnlyStudentList;
+import atas.model.StudentList;
 
 /**
- * Represents a storage for {@link AddressBook}.
+ * Represents a storage for {@link StudentList}.
  */
-public interface AddressBookStorage {
+public interface AtasStorage {
 
     /**
      * Returns the file path of the data file.
      */
-    Path getAddressBookFilePath();
+    Path getStudentListFilePath();
 
     /**
-     * Returns AddressBook data as a {@link ReadOnlyAddressBook}.
+     * Returns AddressBook data as a {@link ReadOnlyStudentList}.
```

Comment:
Should "AddressBook" be refactored to "StudentList"?

Reply:
Thank you, I've missed it out!